docs/documentation/pytest_hardpy.md

```diff
@@ -178,6 +178,51 @@ def test_message():
     set_message("Update message 2", "msg_upd")
 ```
 
+#### run_dialog_box
+
+Displays a dialog box and updates the 'dialog_box' field in the **statestore** database.
+
+    Args:
```

Comment:
The argument is `DialogBoxData`. Required to describe what `DialogBoxData` is.

Reply:
Ready